python_modules/libraries/dagster-fivetran/dagster_fivetran/ops.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Dict
+
+from dagster import Field, In, Noneable, Nothing, op
+from dagster_fivetran.resources import DEFAULT_POLL_INTERVAL
+
+
+@op(
+    ins={"start_after": In(Nothing)},
+    config_schema={
+        "connector_id": Field(
+            str,
+            is_required=True,
+            description="The Fivetran Connector ID that this op will sync. You can retrieve this "
+            'value from the "Setup" tab of a given connector in the Fivetran UI.',
+        ),
+        "poll_interval": Field(
+            float,
+            default_value=DEFAULT_POLL_INTERVAL,
+            description="The time (in seconds) that will be waited between successive polls.",
+        ),
+        "poll_timeout": Field(
+            Noneable(float),
+            default_value=None,
+            description="The maximum time that will waited before this operation is timed out. By "
+            "default, this will never time out.",
+        ),
+    },
+    required_resource_keys={"fivetran"},
+)
+def fivetran_sync_op(context) -> Dict[str, Any]:
```

Comment:
Want to start a discussion in Slack to talk through what our convention should be with respect to suffixing op names with `_op`? It seems like half our integrations do it, and half our integrations don't.

I think we generally do not encourage doing that with non-library ops - e.g. none of the ops in the hacker news jobs have the `_op` suffix.